src/database/index.ts

```diff
@@ -51,19 +51,32 @@ const connectionOptions = (() => {
 let pool: Pool;
 let serverReady = false;
 
-setTimeout(() => {
-  pool = createPool({
-    connectTimeout: 60000,
-    trace: false,
-    ...connectionOptions,
-    typeCast,
-  });
+const mysqlInit = (() => {
+  setTimeout(() => {
```

Comment:
Not sure why this `setTimeout` is here. I know it was there before your PR, testing worked fine without it so not sure what it's meant to accomplish. 
```suggestion
```